Domain/Sia.Shared.Tests/TestDoubles/MockShortCircuit.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Sia.Shared.Extensions.Mediatr;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+
+namespace Sia.Gateway.Tests.TestDoubles
+{
+    internal class MockShortCircuit : HandlerShortCircuit<StubRequest, string>
```

Comment:
Now that I think about it, this should be able to be : HandlerShortCircuit<IRequest<string>, string>

Then we could remove the StubRequest class; we always pass null anyway, so it won't change behavior in the tests, just cleans this file up a bit.

Reply:
Changing it to IRequest gets you this error:  "The type MediatR.IRequest must be convertible to MediatR.IRequest<string> in order to use it as a parameter ..."